front-end/src/app/header/header.component.html

```diff
@@ -1,3 +1,17 @@
-<p>
-  header works!
-</p>
+<div class="container">
+  <nav class="navbar navbar-expand navbar-dark bg-dark">
+    <h3 class="navbar-brand">Networking Assistant</h3>
+    <ul class="navbar-nav" *ngIf="tabs">
+      <li class="nav-item active"><a class="nav-link" [routerLink]="['profile']">Profile</a></li>
```

Comment:
square brackets around url is unnecessary. please remove them.

Reply:
Fixed